charts/ceph-csi-rbd/values.yaml

```diff
@@ -199,8 +199,9 @@ provisioner:
 
   provisioner:
     image:
-      repository: registry.k8s.io/sig-storage/csi-provisioner
-      tag: v3.1.0
+      # TODO: replace with released image version.
```

Comment:
can you reference the PR or commit that added support for this? that makes it possible to verify what release can be used later on

Reply:
and most importantly, in the `# TODO: ` comment, please!